docs/howtos/adapt_vqe.rst

```diff
@@ -8,48 +8,62 @@ This tutorial outlines how the algorithm can be used.
 
 1. We obtain an ``ElectronicStructureProblem`` which we want to solve:
 
->>> from qiskit_nature.second_q.drivers import PySCFDriver
->>> driver = PySCFDriver(atom="H 0 0 0; H 0 0 0.735", basis="sto-3g")
->>> problem = driver.run()
+.. testcode::
 
-2. We setup our ``QubitConverter``:
+    from qiskit_nature.second_q.drivers import PySCFDriver
+    driver = PySCFDriver(atom="H 0 0 0; H 0 0 0.735", basis="sto-3g")
+    problem = driver.run()
 
->>> from qiskit_nature.second_q.mappers import JordanWignerMapper, QubitConverter
->>> converter = QubitConverter(JordanWignerMapper())
+2. We setup our ``QubitMapper``:
+
+.. testcode::
+
+    from qiskit_nature.second_q.mappers import JordanWignerMapper
+    mapper = JordanWignerMapper()
 
 3. We setup our ansatz:
 
->>> from qiskit_nature.second_q.circuit.library import UCCSD, HartreeFock
->>> ansatz = UCCSD()
->>> ansatz.num_particles = problem.num_particles
->>> ansatz.num_spatial_orbitals = problem.num_spatial_orbitals
->>> ansatz.qubit_converter = converter
->>> initial_state = HartreeFock()
->>> initial_state.num_particles = problem.num_particles
->>> initial_state.num_spatial_orbitals = problem.num_spatial_orbitals
->>> initial_state.qubit_converter = converter
->>> ansatz.initial_state = initial_state
+.. testcode::
+
+    from qiskit_nature.second_q.circuit.library import UCCSD, HartreeFock
+    ansatz = UCCSD()
```

Comment:
Similar comment to that of the readme in terms of HF+UCCSD creation

Reply:
Sorry I overlooked it in the existing how-to guide and only updated the newly added ones. Will do this rn